### ISSUE TITLE
CssModules(styles,options)(Component) 

### ISSUE THREAD

**Issue by export-mike** [gajus/react-css-modules#120]
I'd like this ^^^ is this possible? thinking I might just wrap CssModules with another function that exposes this interface so I can do this kind of composition: https://github.com/acdlite/recompose/blob/master/docs/API.md#api

**Comment by ghengeveld** [comment]
Afaik, it already works that way. It's the ES7 decorator syntax.

**Comment by export-mike** [comment]
I've tried it like this with recompose/compose. which is compliant.
![image](https://cloud.githubusercontent.com/assets/2983327/15016366/965e1b8e-1208-11e6-8275-71bb30ebba34.png)

**Comment by export-mike** [comment]
This doesn't work. :(

**Comment by gajus** [comment]
@export-mike It should work, https://github.com/gajus/react-css-modules/blob/e1b4a3da4667a7324a5d6f53f8895be6dc61f7ca/src/index.js#L51.

Have a look at the source code and see if you can find out why it does not work.

**Comment by bmakuh** [comment]
@export-mike I agree that this is a little nicer. I wrote a quick little wrapper that I use in my application:
```
import rcm from 'react-css-modules'

// CSSModules :: {StyleMap, Options} -> ReactComponent -> ReactComponent
export const CSSModules = (styles, opts) => (component) => rcm(component, styles, opts)
```
